.github/workflows/ci.yml

```diff
@@ -0,0 +1,20 @@
+name: GitHub Actions
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js 13.x
```

Comment:
We should use node 12 since it's the latest LTS.

Reply:
ok, changed